examples/run/cuda/seeding_example_cuda.cpp

```diff
@@ -159,12 +170,12 @@ int seq_run(const traccc::seeding_input_config& i_cfg,
             std::cout << "===>>> Event " << event << " <<<===" << std::endl;
 
             // Compare the seeds made on the host and on the device
-            traccc::collection_comparator<traccc::seed> compare_seeds{
+            /* traccc::collection_comparator<traccc::seed> compare_seeds{
                 "seeds", traccc::details::comparator_factory<traccc::seed>{
                              traccc::get_data(spacepoints_per_event),
                              traccc::get_data(spacepoints_per_event)}};
             compare_seeds(vecmem::get_data(seeds),
-                          vecmem::get_data(seeds_cuda));
+                          vecmem::get_data(seeds_cuda)); */
```

Comment:
Doesn't the comparison work between the host- and device-created `alt_seed`-s?

Reply:
The host still functions on `seed` rather than `alt_seed`. I could add code which compares between the two but it seems unnecessary as it would be unused very shortly.

Reply:
Okay, never mind then. You'll re-activate the comparison when the host code is also migrated to the new EDM.